backend/src/game/turn.py

```diff
@@ -0,0 +1,41 @@
+import collections
+
+from backend.src.game import constants
+from backend.src.game.dice.dice import DieValue
+from backend.src.game.locations import Locations
+
+
+def get_dice_count(dice):
+    return collections.Counter(dice)
+
+
+def calculate_victory_points_from_dice(dice_counter):
+    victory_points = 0
+    victory_dice = [DieValue.ONE, DieValue.TWO, DieValue.THREE]
+    for victory_die in victory_dice:
+        die_count = dice_counter[victory_die]
+        leftover_dice = die_count - 3
+        if leftover_dice >= 0:
+            victory_points += (victory_die.value + leftover_dice)
+    return victory_points
+
+
+def calculate_energy_from_dice(dice_counter):
```

Comment:
Can you please add 1 simple test for each of these, energy, attack, heal methods? Maybe even remove the energy method for now since we're not handling that yet